services/stripe-webhook.php

```diff
@@ -426,7 +426,14 @@
 		else
 			$logstr .= "No event ID given.";
 
-		do_action( 'pmpro_webhook_unhandled' );
+/**
+ * Allow hooking into after a webhook has been run but was not handled.
+ *
+ * @since TBD
+ *
+ * @param string $gateway The gateway the webhook was not handled for.
+ */
+do_action( 'pmpro_webhook_unhandled', PMPRO_DOING_WEBHOOK );
```

Comment:
```suggestion
	/**
	 * Allow hooking into after a webhook has been run but was not handled.
	 *
	 * @since TBD
	 *
	 * @param string $gateway The gateway the webhook was not handled for.
	 */
	do_action( 'pmpro_webhook_unhandled', PMPRO_DOING_WEBHOOK );
	
```